src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
  * added `Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface`
  * added `Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface` as argument to `HttpKernel`
  * added `Symfony\Component\HttpKernel\Controller\ArgumentResolver`
+ * added the `kernel.controller_arguments` event, triggered after controller arguments have been resolved
```

Comment:
This sentence sounds more like a commit message which is is trying to be as short as possible. If you would change it slightly, it would sound a lot better:

* added the `kernel.controller_arguments` event, **which is** triggered after **the** controller arguments have been resolved